synchronizer/synchronizer.go

```diff
@@ -123,6 +125,37 @@ func NewSynchronizer(
 		syncBlockProtection:           syncBlockProtection,
 		halter:                        syncCommon.NewCriticalErrorHalt(eventLog, 5*time.Second), //nolint:gomnd
 	}
+	if cfg.L1BlockCheck.Enable {
+		log.Infof("L1BlockChecker enabled: %s", cfg.L1BlockCheck.String())
+		l1requester, ok := res.etherMan.(l1_check_block.L1Requester)
```

Comment:
I think this wrapper does not make any sense. You could just use the etherman and specify just the method that you eed in the interface

Reply:
totally agree! good catch!